MediathekView 12 doesn't start

I'm running Fedora 24 (x86-64) and MediathekView 11 starts fine.

```
$ java -version
openjdk version "1.8.0_101"
OpenJDK Runtime Environment (build 1.8.0_101-b14)
OpenJDK 64-Bit Server VM (build 25.101-b14, mixed mode)
```

MediathekView 12 gives the following error:

```
$ ./MediathekView__Linux.sh 
Exception in thread "main" java.lang.NoClassDefFoundError: javafx/collections/FXCollections
        at mSearch.tool.SysMsg.<clinit>(SysMsg.java:27)
        at mediathek.Main.main(Main.java:94)
Caused by: java.lang.ClassNotFoundException: javafx.collections.FXCollections
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 2 more
```

--- a/.gitignore
+++ b/.gitignore
@@ -2,6 +2,7 @@ build
 dist/javadoc
 /.gradle/
 /bin/
+/.checkstyle
 .classpath
 .project
 .settings/

--- a/src/mediathek/Main.java
+++ b/src/mediathek/Main.java
@@ -60,6 +60,27 @@ public class Main {
         } catch (Exception ignored) {
         }
     }
+    
+    /**
+     * Tests if javafx is in the classpath by loading a well known class.
+     */
+    private static void testForJavaFx() {
+        try {
+            Class.forName("javafx.collections.FXCollections");
+        } catch (ClassNotFoundException e) {
+            final String[] javaFxErrorMsg = {
+                    "===========================================",
+                    "JavaFX wurde nicht im klassenpfad gefunden.",
+                    "Stellen Sie sicher, dass Sie ein Java JRE ab Version 8 benutzen.",
+                    "Falls Sie Linux nutzen, installieren Sie das openjfx-Paket ihres Package-Managers,",
+                    "oder nutzen Sie eine eigene JRE-Installation.",
+                    "===========================================" };
+            SysMsg.sysMsg(javaFxErrorMsg);
+
+            System.exit(1);
+        }
+
+    }
 
     /*
      * Aufruf:
@@ -166,6 +187,7 @@ public class Main {
 
             case GUI:
                 EventQueue.invokeLater(() -> {
+                    testForJavaFx();
                     //JavaFX stuff
                     Platform.setImplicitExit(false);
 

--- a/src/mediathek/Main.java
+++ b/src/mediathek/Main.java
@@ -60,26 +60,23 @@ public class Main {
         } catch (Exception ignored) {
         }
     }
-    
+
     /**
      * Tests if javafx is in the classpath by loading a well known class.
      */
     private static void testForJavaFx() {
         try {
-            Class.forName("javafx.collections.FXCollections");
+            Class.forName("javafx.application.Platform");
         } catch (ClassNotFoundException e) {
-            final String[] javaFxErrorMsg = {
-                    "===========================================",
-                    "JavaFX wurde nicht im klassenpfad gefunden.",
-                    "Stellen Sie sicher, dass Sie ein Java JRE ab Version 8 benutzen.",
-                    "Falls Sie Linux nutzen, installieren Sie das openjfx-Paket ihres Package-Managers,",
-                    "oder nutzen Sie eine eigene JRE-Installation.",
-                    "===========================================" };
-            SysMsg.sysMsg(javaFxErrorMsg);
+            System.out.println("===========================================");
+            System.out.println("JavaFX wurde nicht im klassenpfad gefunden.");
+            System.out.println("Stellen Sie sicher, dass Sie ein Java JRE ab Version 8 benutzen.");
+            System.out.println("Falls Sie Linux nutzen, installieren Sie das openjfx-Paket ihres Package-Managers,");
+            System.out.println("oder nutzen Sie eine eigene JRE-Installation.");
+            System.out.println("===========================================");
 
             System.exit(1);
         }
-
     }
 
     /*
@@ -97,6 +94,7 @@ public class Main {
      * @param args the command line arguments
      */
     public static void main(final String args[]) {
+    	testForJavaFx();
         StartupMode state = StartupMode.GUI;
 
         try {
@@ -187,7 +185,6 @@ public class Main {
 
             case GUI:
                 EventQueue.invokeLater(() -> {
-                    testForJavaFx();
                     //JavaFX stuff
                     Platform.setImplicitExit(false);
 

